src/main/scala/com/fulcrumgenomics/bam/SamRecordClipper.scala

```diff
@@ -327,23 +327,23 @@ class SamRecordClipper(val mode: ClippingMode, val autoClipAttributes: Boolean)
   /** Returns the number of bases extending past the mate end for FR pairs including any soft-clipped bases, zero otherwise.
     *
     * @param rec the record to examine
-    * @param mateUnclippedStart the smallest mapped genomic coordinate considering clipping
-    * @param mateUnclippedEnd the largest mapped genomic coordinate considering clipping
+    * @param mateUnSoftClippedStart the smallest mapped genomic coordinate considering *ONLY* soft-clipping
+    * @param mateUnSoftClippedEnd the largest mapped genomic coordinate considering *ONLY* soft-clipping
```

Comment:
I don't love the new names for these but really couldn't think of anything better.